packages/search/src/elasticsearch/utils.ts

```diff
@@ -409,18 +421,49 @@ export function isValidOperationHistory(body: IBirthCompositionBody) {
       VALIDATED_STATUS,
       WAITING_VALIDATION_STATUS
     ],
-    [CERTIFIED_STATUS]: [REGISTERED_STATUS, CERTIFIED_STATUS]
+    [CERTIFIED_STATUS]: [REGISTERED_STATUS, CERTIFIED_STATUS],
+    [REQUESTED_CORRECTION_STATUS]: [REGISTERED_STATUS, CERTIFIED_STATUS]
```

Comment:
I have added this entry in `validStatusMapping`; that means a correction request can only take place after the application is `REGISTERED` or `CERTIFIED`. Not sure user can request correction for the application which is already in `REQUESTED_CORRECTION` status